src/Task1/SelectGenre.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import "./selectGenre.css";
+const SelectGenre = () => {
+  const [show, setShow] = useState("");
+  const [clip, setClippath] = useState("polygon(50% 75%, 0 25%, 100% 25%)");
+  return (
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      <div
+        id="list1"
+        className={"dropdown-check-list " + show}
+        tabindex="100"
+        style={{ width: "400px", backgroundColor: "#232323", color: "#FFFFFF" }}
+      >
+        <span
+          onClick={() => {
+            if (show === "visible") {
+              setShow("");
+              setClippath("polygon(50% 75%, 0 25%, 100% 25%)");
+            } else {
+              setShow("visible");
+              setClippath("polygon(50% 25%, 0 75%, 100% 75%)");
+            }
+          }}
+          className="anchor"
+          style={{
+            width: "96%",
+            backgroundColor: "#323232",
+            fontSize: "20px",
+          }}
+        >
+          Select Genre
+          <span style={{ clipPath: clip }} className="arrow">
+            ok
+          </span>
+        </span>
+
+        <ul className="items" style={{ marginTop: "20px" }}>
+          <li>
+            <label className="container">
+              Crime
+              <input type="checkbox" />
+              <span class="checkmark"></span>
```

Comment:
Most probably, from the semantics standpoint, these options should actually be buttons, not check-boxes. To track which option is selected, you can use React's state with custom styling based on the current state value.

Reply:
Yes, I used button

Reply:
Sorry for this comment. I was actually wrong. I didn't pay attention to the fact that multiple options can be selected in a check-box way. These should not be buttons. Your initial approach with labels and checkbox inputs was better. Can you please revert your code to that state?

Reply:
I used button already. but I can implement using button also. So should I change anyway?

Reply:
> but I can implement using _button_ also

Do you mean using checkboxes? If yes, then please do it. As you did initially. Sorry for the inconveniences caused by mistake.

Reply:
No, I used button, ok, I will do it with checkbox again